bot/skills/kozula.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import xml.etree.ElementTree as ET
+import requests
+from mode import cleanup_queue_update
+
+from telegram import Update
+from telegram.ext import Updater, CommandHandler, CallbackContext
+
+logger = logging.getLogger(__name__)
+
+KOZULA_RATE = 600_000
+CBR_URL = "https://cbr.ru/scripts/XML_daily.asp"
+
+
+def add_kozula(upd: Updater, handlers_group: int):
+    logger.info("registering tree handlers")
+    dp = upd.dispatcher
+    dp.add_handler(CommandHandler("kozula", kozula, run_async=True), handlers_group)
+
+
+def kozula(update: Update, context: CallbackContext):
+    try:
+        request = requests.get(CBR_URL)
```

Comment:
To be honest, looks like part with an HTTP request could be extracted. And it should call through some cache.